simulation/src/wildlife/bird.rs

```diff
@@ -0,0 +1,28 @@
+use crate::map_dynamic::Itinerary;
+use crate::physics::Speed;
+use crate::transportation::bird_mob::BirdMob;
+use crate::utils::rand_provider::RandProvider;
+use crate::Simulation;
+use crate::{BirdEnt, BirdID};
+use geom::{Transform, Vec3};
+
```

Comment:
I think all of the code can belong in this file. The transportation module is more for things that belong on different networks such as Cars, Trains and Pedestrians.